### DISCUSSION TITLE
Halley Comet Ancient Data

### DISCUSSION THREAD

**Discussion by MonteShaffer** [Stellarium/stellarium#2422] [Q&A]
I am new to Stellarium.

I loaded a middle east location and put in the date of Aug 6, 87 BC and I don't see Halley's comet.

WIKIPEDIA has a list of some of the ancient times.  How can I see them in Stellarium.
https://en.wikipedia.org/wiki/Halley%27s_Comet

I also don't understand how a comet can have 74-79 year return.  Are not orbits consist, like every 75 years?

**Comment by gzotti** [comment]
> I am new to Stellarium.
> 
> I loaded a middle east location and put in the date of Aug 6, 87 BC and I don't see Halley's comet.
> 
> WIKIPEDIA has a list of some of the ancient times. How can I see them in Stellarium. https://en.wikipedia.org/wiki/Halley%27s_Comet

Stellarium User Guide section 13.7 has the answer. You can load a data file with elements for historical apparitions.

> I also don't understand how a comet can have 74-79 year return. Are not orbits consist, like every 75 years?

No. Read about "Orbit and origin" in the article you referenced, and https://en.wikipedia.org/wiki/Perturbation_(astronomy) linked therein.

**Reply by Atque** [reply]
>You can load a data file with elements for historical apparitions.

Where are the orbital elements in `ssystem_1000comets.ini` taken from? I highly doubt that the comets of pre-1600 would have accurate orbits in the slightest. Or are there actual sources of e.g. the Great comet of 1337, that say something like "near the Pole star, a comet appeared in mid-July"?

**Reply by gzotti** [reply]
I have found the list in 2014 at http://spider.seds.org/spider/Comets/comets.data and converted it into Stellarium's format. SOHO comets were excluded. The list was apparently made in 1996, just before Hyakutake and Hale-Bopp and the new comet designation system.

Elements for Halley taken directly from Yeomans&Kiang MNRAS.197..643 (1981). H10 (missing there) from Mucke: Komet Halley als säkulare Himmelserscheinung. Die Sterne 61-5/6 (1986) and extrapolated before -239

You may doubt accuracy, but the original authors hopefully had some insights on what they were doing. Corrections or alternatives (with source) welcome.

**Reply by MonteShaffer** [reply]
Hello,

I am getting a 404 on the link for comets.data ... https://web.archive.org/web/20180801000000*/http://spider.seds.org/spider/Comets/comets.data

I found a few harvard articles that have tables with the updated "historic" predictions.

https://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?1982JBAA...92..209B&defaultprint=YES&filetype=.pdf
-619 Oct 5  ... Oct 5, 620 BC
 -11 Oct 8

https://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?1981MNRAS.197..633Y&defaultprint=YES&filetype=.pdf
-616 Jul 28 ... Jul 28, 617 BC
 -11 Oct 10

I found another article that refers to Halley's and two other Chinese comets:
https://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?1991QJRAS..32..389H&defaultprint=YES&filetype=.pdf
Ho Peng Yoke (1962)
	61		12 BC Aug 26 ...  po (comet)
	63		5 BC (March 9 - April 6) ... sui (tailed comet)
	64		4 BC April ... po (comet)

And I found a parser for stellarium:
https://projects.familie-steinel.de/stellarium-comet-jpl/

I can certainly offer any comet data I create to the open-source community.  I can do math/programming so I can figure out what some of the tables mean.  I just need a bit more direction on how to convert a research paper with parameters that are updating for Halley's; e.g.,

[Halley's Comet: AD 1986 to 2647 BC - NASA/ADShttps://adsabs.harvard.edu › full](https://adsabs.harvard.edu/full/1982JBAA...92..209B)
by JL Brady · 1982 · Cited by 14 — 1982, 92, 5 Halley's Comet 209 Halley's Comet: AD 1986 to 2647 BC JOSEPH L. ... With this evidence, Halley predicted that the comet of 1682 would

How to take his tables and show multiple passes of Halley's comet?  Maybe multiple versions of Halley's based on the different results?  If Halley's comes close to a planet on its inward path it will change some of its orbital parameters slightly.  It appears the 1982 paper questions some of the Yeomans(1981) methods.

How to find the ancient Chinese records and implement them as a data file?  Where are they?

Anyway, if you could share that comets.data file you have that would certainly help.  Any suggestions would be appreciated.

Cheers,
